tests/test_signing.py

```diff
@@ -0,0 +1,90 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+
+import tempfile
+from collections.abc import Mapping
+from pathlib import Path
+
+import pytest
+
+from mkosi.run import find_binary, run
+
+from . import Image, ImageConfig
+
+pytestmark = pytest.mark.integration
+
+
+def test_signing_checksums_with_sop(config: ImageConfig) -> None:
+    if find_binary("sqop", root=config.tools) is None:
+        pytest.skip("Needs `sqop` binary in PATH to perform sop tests.")
```

Comment:
Nit: let's not do backticks, since they can evaluate in a shell and people tend to copy stuff around, also, an empty line after this would be nice.
```suggestion
        pytest.skip("Needs 'sqop' binary in PATH to perform sop tests.")
        
```